script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -196,14 +198,14 @@ Following her gaze,
 [sha:25167041b41723711b35f94783944ae9ede4646e]{
 -- Page 54, Offset 29645.
 -- 「……俺は、自分を確かめに来ただけなんだ。
-"...I just came to do some soul searching.
+"I just came to do some soul searching...
 }
 [sha:17cceb2593236637ad76d73ca29ae8ae905ac80e]{
 -- Page 54, Offset 29646.
 -- それももう、半分ぐらい答えは出た。結局、俺は」
 // Fairly liberal, literally more like 'in regards to that, I have only arrived at half an answer'
 // Referenced later in file
-Though, I've still only found half an answer.
+Though, I've still only found half the answer.
```

Comment:
"Half the answer" is a far, far, far more common expression.

Reply:
They're surprisingly close actually. I guess my bent towards 'an' here is that it implies a more vague answer space, rather than  a single definite answer. 
![2022-06-25-161752_1570x571_scrot](https://user-images.githubusercontent.com/2476799/175789326-47b79302-a174-4f60-87b4-c6040c1cdfb1.png)

Reply:
That an answer exists as a possibility of being found in Shiki's space implies the existence of a question he's trying to answer:

I think being 'vague' about actually confuddles the point:

The Question: what is it he should do?
The Answer (1/2): he should go confront Roa
The Answer (2/2): because he wants to be with Arcueid

So, I think when you phrase it as "an answer" it feels as though he's just pondering "something", which isn't the point of this scene at all.

Reply:
Fair enough.